boards/arm/96b_aerocore2/96b_aerocore2.dts

```diff
@@ -36,6 +36,7 @@
 	aliases {
 		led0 = &yellow_led_1;
 		led1 = &blue_led_2;
+		usbd0 = &usbotg_fs;
```

Comment:
DT_ALIAS(usbd0) is not a reason, usbd0 is common aliases for the USB device controller node and is used in all the samples overlay files.

Reply:
This aliases actually has no effect, I think it is not possible to use them like this. app.overlay below works just as well.
```
/ {                                                                                           
        idonotlikealiases0 {                                                                               
                cdc_acm_uart0 {                                                               
                        compatible = "zephyr,cdc-acm-uart";                                   
                        label = "CDC_ACM_0";                                                  
                };                                                                            
        };                                                                                    
}; 
```

Reply:
> This aliases actually has no effect, I think it is not possible to use them like this. app.overlay below works just as well.

Right, so this is creating a node `/idonotlikealiases0/cdc_acm_uart0`, which is not a child of any USB controller. That is the problem that you can avoid by implementing the `bus: usb` + `on-bus: usb` suggestion I posted about above:

https://github.com/zephyrproject-rtos/zephyr/pull/37085#discussion_r682974127

Unfortunately, due to the way the DT tooling works right now, you cannot use `on-bus: usb` to force the node to be on that bus, because that is also used to make multiple bindings for the same compatible on different buses (like separate bindings for SPI and I2C for the same compatible).

However, you can do something like this:

```
#define CDC_ACM_DT_DEVICE_DEFINE(idx)					\
	BUILD_ASSERT(DT_INST_ON_BUS(idx, usb),				\
		     "node " DT_NODE_PATH(DT_DRV_INST(idx))		\
		     " is not on a USB bus");				\
	...
```

Than you would know that the nodes are all on the right sort of bus, or get a sensible error, at build time.

Reply:
that is good, I took it over, but changed assert error, thank you